src/components/headMetaTags.js

```diff
@@ -16,14 +41,17 @@ class HeadMetaTags extends Component {
           ]}
           title={title}
         />
+        {schema &&
+        <Helmet script={[schema]} />
+        }
```

Comment:
Do we need a separate `<Helmet>` component for this? Can't it just be included in the other one? React won't pass a prop to a component if the value is undefined.